### PR TITLE
[logging][easy] move security logs to new structure logging macro

### PR DESCRIPTION
**Pull request by mimoo**
closes https://github.com/libra/libra/issues/5612

### CODE REVIEW

File: consensus/src/epoch_manager.rs [hunk 1]
```diff
@@ -398,7 +398,7 @@ impl EpochManager {
                 .context("[EpochManager] Verify event")
                 .map_err(|err|
                         // security log + return the error
-                        {send_struct_log!(security_log(security_events::CONSENSUS_INVALID_MESSAGE)
+                        {sl_error!(security_log(security_events::CONSENSUS_INVALID_MESSAGE)
                             .data("from_peer", &peer_id)
                             .data_display("error", &err)
                             .data("event", &unverified_event)
```
Comment (gregnazario) [inline consensus/src/epoch_manager.rs:404]: Tbh, this formatting seems weird, but it was already there.  Was this the autoformatter?

Comment (mimoo) [reply consensus/src/epoch_manager.rs]: I made it nicer, that's one problem with rustfmt: it seems to allow different formatting to be valid : ( (unlike gofmt)

Comment (gregnazario) [reply consensus/src/epoch_manager.rs]: Rust format doesn't seem to format nicely under some very specific nesting conditions.  I notice it doesn't like macros and we sometimes use macros for large blocks

**Pull request closed without merging**